configs/toolchain-llvm.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "toolchain-llvm-18",

```

Comment:
this should remain as `toolchain-llvm-18`, same with file name.

That way when 20 is added, we can keep `toolchain-llvm-18`.